### DISCUSSION TITLE
Animation Montage stays in place

### DISCUSSION THREAD

**Discussion by yukioshima** [Sixze/ALS-Refactored#300] [Q&A]
I am trying to implement a jump over a valley to transition from ALS to "ALS-Refactored", but the Animation Montage that can be used in the DefaultSlot for normal ALS is stuck in place for "ALS-refactored".
I have confirmed that using the "Locomotion.PostLocomotion" slot will play the animation, but I am unable to solve this problem.
I would like to know if anyone has a solution to this difference.

https://user-images.githubusercontent.com/30016097/236628157-83fc4ccc-016d-48db-ad68-8eb1b0cb3f11.mp4

**Comment by murataka** [comment]
check if the jump animation has root motion , check if it is set to enabled in animation sequence.
